subprojects/javascript/src/main/java/org/gradle/plugins/javascript/coffeescript/CoffeeScriptCompile.java

```diff
@@ -47,6 +49,7 @@ protected WorkerProcessFactory getWorkerProcessBuilderFactory() {
         throw new UnsupportedOperationException();
     }
 
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Does this care about names at all?

Reply:
Well, if I rename a file then the output file will be in a different location. So I guess this needs to be relative given the structure of the sources is mirrored in the compiled files.